core/templates/pages/contributor-dashboard-admin-page/services/contributor-dashboard-admin-backend-api.service.ts

```diff
@@ -135,7 +135,7 @@ export class ContributorDashboardAdminBackendApiService {
     return new Promise((resolve, reject) => {
       this.http.delete<void>(url, {
         params
```

Comment:
Can you put `url, { params } as Object` in this line? That would read better.

Alternatively something like https://github.com/oppia/oppia/pull/16597/files?file-filters%5B%5D=.py&file-filters%5B%5D=.ts&owned-by%5B%5D=seanlip&show-viewed-files=true#diff-004bcdf23f8fc0f4dad66c09ecdec309358013e4f6ce48ceaf01706436008d5bR189 would work.

Reply:
Done